### DISCUSSION TITLE
Kusto Query Language (KQL) and Bicep support

### DISCUSSION THREAD

**Discussion by ghost** [massCodeIO/massCode#313] [Ideas]
Hi,

Just tried the app and it looks to be exactly what I am looking for but misses one feature I really would like, support for KQL.
Would it be possible to add syntax support for that too?

https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/

https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview?tabs=bicep

Thanks!

**Comment by antonreshetov** [comment]
@eplusg Added in #324 & #325. Will be available with the new release. 

![SCR-20230110-bu8](https://user-images.githubusercontent.com/17480987/211469532-21130f0d-e4e5-4b78-92ed-0fdb25a9e6f6.png)
![SCR-20230110-bud](https://user-images.githubusercontent.com/17480987/211469538-a2972035-80e0-4ecf-9b97-0ef9d91d6711.png)
